src/Input/ChildCommandFactory.php

```diff
@@ -28,7 +28,7 @@
 final class ChildCommandFactory
 {
     public function __construct(
-        private readonly string $phpExecutable,
+        public readonly string $phpExecutable,
```

Comment:
To be able to pass the PHP executable to `PhpSubprocess` we need to expose it, instead of passing it as part of the created command